### PR TITLE
Convert our actions to our tools

### PR DESCRIPTION
**Pull request by clementbiron**
Difficulty estimation : 🟧⬜⬜

### CODE REVIEW

File: assets/css/style.scss [hunk 1]
```diff
@@ -1,9 +1,5 @@
----
-# this ensures Jekyll reads the file to be transformed into CSS later
-# only Main files contain this front matter, not partials.
----
-
-//Reset CSS
+--- # this ensures Jekyll reads the file to be transformed into CSS later # only Main files contain this front matter,
```
Comment (Ndpnt) [inline assets/css/style.scss]: Why this changes?

Comment (clementbiron) [reply assets/css/style.scss]: no idea ! 🤷‍♂️

**Review by Ndpnt — APPROVED** [review]
